build/filter-dev.properties

```diff
@@ -7,11 +7,10 @@ cassandra.port=9160
 cassandra.clusterName=gooru-cassandra-qa
 cassandra.keyspace.insights=insights_qa
 cassandra.keyspace.search=insights_qa
-redis.server.host=ec2-54-176-214-187.us-west-1.compute.amazonaws.com
 dataCentre=datacenter1
 
 #redis
-redis.server.host=ec2-54-176-214-187.us-west-1.compute.amazonaws.com
+redis.server.host=ec2-54-176-76-27.us-west-1.compute.amazonaws.com
```

Comment:
@kumaraswamyS 
please remove all these server ips from filter and commit. we should not have this here...